monetize.gemspec

```diff
@@ -1,34 +1,31 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'monetize/version'
-require 'English'
+require "monetize/version"
 
-Gem::Specification.new do |spec|
-  spec.name          = 'monetize'
-  spec.version       = Monetize::VERSION
-  spec.authors       = ['Shane Emmons', 'Anthony Dmitriyev']
-  spec.email         = ['shane@emmons.io', 'anthony.dmitriyev@gmail.com']
-  spec.description   = 'A library for converting various objects into `Money` objects.'
-  spec.summary       = 'A library for converting various objects into `Money` objects.'
-  spec.homepage      = 'https://github.com/RubyMoney/monetize'
-  spec.license       = 'MIT'
+Gem::Specification.new do |s|
+  s.name        = "monetize"
```

Comment:
I just thought it would be easier for us if all the gems in this org keep the same structure 😊